ci/jenkinsfiles/build.groovy

```diff
@@ -17,46 +17,103 @@
 *     Abdoul BA <aba@nuxeo.com>
 */
 
+/* Using a version specifier, such as branch, tag, etc */
+@Library('nuxeo-napps-tools@improv-NXBT-3447-napps-shared-libs') _
+
```

Comment:
should be changed to the last pushed version, right? on Aspera we have [0.0.4](https://github.com/nuxeo/nuxeo-aspera-connector/commit/67dc359d9192386d5efc45ed0421ac464e65ec59#diff-e6ffa5dc854b843b3ee3c3c28f8eae2f436c2df2b1ca299cca1fa5982e390cf8R24)

Reply:
I think we should squash this commit with the first one, on CI/CD work